### ISSUE TITLE
Add facebook signup

### ISSUE THREAD

**Issue by begedin** [code-corps/code-corps-ember#35]
#34  needs to be merged before this can be done, since there is overlap

**Comment by begedin** [comment]
Blocked by #2

**Comment by begedin** [comment]
Some information that might be useful:

**Comment by JoshSmith** [comment]
Closing because we can readdress later if needed. This is very much not a priority.
